src/Mumble.proto

```diff
@@ -63,7 +63,7 @@ message Ping {
 // Sent by the server when it rejects the user connection.
 message Reject {
 	enum RejectType {
-		// TODO ??
+		// The rejection reason is unknown (murmur will never send this value).
```

Comment:
If the stock client interprets this as "Unknown" then we shouldn't say the server never sends it. We will likely need it for backwards compatibility in the future and if someone implements theprotocol they should handle it correctly.